code/modules/projectiles/guns/projectile/pistol.dm

```diff
@@ -86,4 +86,36 @@
 	icon_state = "[initial(icon_state)][silenced ? "-silencer" : ""][chambered ? "" : "-e"]"
 	return
 
-
+/obj/item/weapon/gun/projectile/handgun //mime fingergun 
+	name = "hand-gun"
+	desc = "This is a stickup!"
+	icon_state = "handgun"
+	inhand_states = null
+	ammo_type = "/obj/item/ammo_casing/invisible"
+	mag_type = "/obj/item/ammo_storage/magazine/invisible"
+	cant_drop = 1
```

Comment:
Because it looks clearer. You can instantly tell something is a bool if it gets set to TRUE or FALSE instead of 1 or 0.

    myvar = 0

is more ambiguous than

    myvar = FALSE